src/infohud.asm

```diff
@@ -436,6 +440,35 @@ ih_update_hud_code:
     RTL
 }
 
+ih_update_hud_early:
+{
+    PHA
+    PHX
+    PHY
+
+    ; calculate lag frames
+    LDA !ram_realtime_room : SEC : SBC !ram_transition_counter : STA !ram_last_room_lag
+
+    LDA !ram_gametime_room : STA !ram_last_gametime_room
+    LDA !ram_realtime_room : STA !ram_last_realtime_room
+
+    ; save temp variables
+    LDA $12 : PHA
+    LDA $14 : PHA
+
+    ; Update HUD
+    JSL ih_update_hud_code
+
+    ; restore temp variables
+    PLA : STA $14
+    PLA : STA $12
+
+    ; Run standard code and return
+    PLY
+    PLX
+    PLA
+}
+
 ih_hud_code:
 {
     %ai16()
```

Comment:
About 50 lines down, there is code to print hp.  Can you add a line to write a space over the decimal at 7EC690?